docs/std-archive.md

```diff
@@ -0,0 +1,42 @@
+# archive
+
+`archive` library contains helper functions that related to outputting data into
+certain formats of archive.
+
+## `tar <: tuple`
+
+`tar` returns a tuple that contains functions related to the format `tar`. The
+attributes are the following:
+
+### `tar(data <: dict) <: array_of_bytes`
+
+It takes the dictionary `data` and returns an array of bytes representing `data`
+which has been compressed to the `tar` format.
+
+**Important note**: `data` itself has to be a dictionary whose keys are all
+`string`.
+
+Usage:
+
+| example | equals |
+|:-|:-|
+| `//archive.tar.tar({"lightsaber": "lightsaber noises"})` | `lightsaber0000600000000000000000000000002100000000000011235 0ustar0000000000000000lightsaber noises` |
```

Comment:
Depending on how long the output tar string is, maybe adding some spaces in the markdown might make it more readable in .md form? Github does line wrapping but at least it'll print nicely when viewing it from an IDE
```suggestion
| example                                                                            | equals                                                   |
|------------------------------------------------------|--------------------------------------------------------|
| `//archive.tar.tar({"lightsaber": "lightsaber noises"})` | `lightsaber00006000000000000000000000000021000000000000112350ustar0000000000000000lightsaber noises` |
```

Reply:
the space is part of the output, removing it might mean a different object is being compressed